symbulate/distributions.py

```diff
@@ -363,6 +363,62 @@ def __init__(self, a, b, scale=None):
     def draw(self):
         return np.random.beta(self.a, self.b)
 
+class t(Distribution):
```

Comment:
Avoid calling this t. (It's very possible the user will have a variable called t already.) Let's call it StudentT.